### DISCUSSION TITLE
Make a release

### DISCUSSION THREAD

**Discussion by XVilka** [open-watcom/open-watcom-v2#487] [Ideas]
Lack of the releases seriously limits the availability in Linux/BSD distributions.
Would be awesome at some point have it packaged and supersede the old OpenWatcom 1.9.

**Comment by rdos314** [comment]
I agree with this. At the moment I've put up a few working builds that I did from source, but a full release with installer and everything would be nice.

**Comment by NCommander** [comment]
Is the code for the installer available? I'd like to be able to at least have a packaged binary that I can burn to a CD for installing on legacy machines vs. having to download a snapshot and setup it up step by step (and how to build said installer).

**Comment by jmalak** [comment]
Yes installer program (setupgui project) is in OW source tree subdirectory bld/setupgui
Build stuff for all installers ( include all files into executable) is in distrib/ow subdirectory.

**Comment by jmalak** [comment]
Installer program for all platforms are build by regular build.
Build of all installers can be done by running command `builder` in directory `distrib/ow`
It can be run only if all documentation for all targets are builded OK
For test purpose (without documentation) it can be build by commands
`builder missing`
`builder`

**Comment by rdos314** [comment]
I think the main problem is to build the documentation. It requires a set of very exotic tools so I don't build the documentation, and without a full build, the installer cannot run.

**Comment by jmalak** [comment]
I can create full set of documentation for you and put on some ftp server if necessary.

**Comment by zoomosis** [comment]
Add me to the list of people who'd like to see a release version.
It would potentially enable Watcom C++ to be in the Ubuntu repos for example, installed simply with "apt-get install openwatcom" or similar.
It could also be made available in the FreeBSD Ports tree, with the Linux binaries running in FreeBSD's Linux subsystem.

**Comment by XVilka** [comment]
See also:
- https://repology.org/project/openwatcom/versions
- https://repology.org/project/openwatcom-v2/versions

**Comment by jmalak** [comment]
Anyway it needs some maintainer to create appropriate package by regular scripts and probably discuse where to locate(install) OW by default. What is used now was derived from QNX installation many years ago.
Installer can be build without documentation (it create missing blank files) or we can modify installer script for linux host to not include these "specific" documentation formats.

**Comment by jmalak** [comment]
installers for all hosts are now build regulary.

**Comment by XVilka** [comment]
I noticed there is a milestone, does it correspond to the release target? https://github.com/open-watcom/open-watcom-v2/milestone/1

**Comment by jmalak** [comment]
No, it is mistake. No milestone is currently planned.
